tests/conn_can/Makefile

```diff
@@ -11,6 +11,7 @@ USEMODULE += can_isotp
 USEMODULE += conn_can_isotp_multi
 USEMODULE += can_pm
 USEMODULE += can_trx
+USEMODULE += auto_init_can
```

Comment:
But you enable `auto_init_can` here.

Reply:
That's in `tests/conn_can/Makefile`. 
It's because we removed the dependency from Makefile.dep. To prevent breaking conn_can, we had to explicitly enable CAN in that app.
That's what `77bfde` : "tests/conn_can: make auto_init_can explicit" does.